src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.properties

```diff
@@ -0,0 +1,63 @@
+approvedScripts=Approved scripts
+
+# approved scripts
+metadataGatheringDisabled=The metadata gathering was disabled. \
+  <p>You can re-enable it permanently by removing the system property: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.metadataGathering</code> </p>\
+  <p>or temporarily (until restart) by running this in the Script Console: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.METADATA_GATHERING = true</code></p>
+
+## table headers
+approvedHash=Hash
+approvedHash_tooltip=The hash could be seen as a signature of the script. It's used as an identifier to reference the content of the script.
```

Comment:
I'm not a fan of calling a hash a signature.

Reply:
Any other proposal?